Alerting: Enable Alert rule `severity` tag to override VictorOps Severity setting (#29392)
grafana/grafana: docs/sources/alerting/notifications.md

This behavior will become the default in a future version of Grafana.

> Using `dedup_key` tag will override Grafana generated `dedup_key` with a custom key.

### VictorOps

> To set up VictorOps, provide the URL from the Grafana Integration and substitute `$routing_key` with the routing key.

Reviewer: ```suggestion
To configure VictorOps, provide the URL from the Grafana Integration and substitute `$routing_key` with a valid key.
```

> Setting | Description

Reviewer: Do we need a table for just one item?

Author: Good point, no

---------- | -----------

> Auto resolve incidents | Resolve incidents in VictorOps once the alert goes back to ok

Reviewer 2: ```suggestion
Auto resolve incidents | Resolve incidents in VictorOps once the alert goes back to ok.
```

Reviewer 2: Should "OK" be capitalized? Please match whatever it says in VictorOps.

> Auto resolve incidents | Resolve incidents in VictorOps once the alert goes back to ok

Author: ```suggestion
Auto resolve incidents | Resolve incidents in VictorOps once the alert goes back to `OK`.
```

> >**Note:** The tag `Severity` has special meaning in the [VictorOps Incident Fields](https://help.victorops.com/knowledge-base/incident-fields-glossary/). If an alert panel defines this tag key, then it is transposed to the root of the event sent to VictorOps.

Reviewer 2: Is "transposed" the right word? If something is transposed, then it switches places with something else.

Perhaps you mean appended, which is adding it to the end?

Author: copied that language from [here](https://github.com/grafana/grafana/blame/460883d0b291a5e497a07e6efd7c8ef88a507fa0/docs/sources/alerting/notifications.md#L125)

It is transposed; it replaces a field in the root of the event. Let me know if I should change this wording and the other please!

Reviewer 2: In that case, It needs to be something like

transposed with the root

or

it replaces the root 

I recommend the second option, as simpler words are easier for not-native English speakers to understand.

Author: ```suggestion
>**Note:** The tag `Severity` has special meaning in the [VictorOps Incident Fields](https://help.victorops.com/knowledge-base/incident-fields-glossary/). If an alert panel defines this tag key, then it replaces the `message_type` in the root of the event sent to VictorOps.
```

> >**Note:** The tag `Severity` has special meaning in the [VictorOps Incident Fields](https://help.victorops.com/knowledge-base/incident-fields-glossary/). If an alert panel defines this tag key, then it replaces the `message_type` in the root of the event sent to VictorOps.

Reviewer: ```suggestion
>**Note:** The tag `Severity` has special meaning in the [VictorOps Incident Fields](https://help.victorops.com/knowledge-base/incident-fields-glossary/). If an alert panel defines this key, then it replaces the `message_type` in the root of the event sent to VictorOps.
```

> **Note:** The `state` tag overrides the current alert state inside the `custom_details` payload.

### VictorOps

To configure VictorOps, provide the URL from the Grafana Integration and substitute `$routing_key` with a valid key.

> >**Note:** The tag `Severity` has special meaning in the [VictorOps Incident Fields](https://help.victorops.com/knowledge-base/incident-fields-glossary/). If an alert panel defines this key, then it replaces the `message_type` in the root of the event sent to VictorOps.

Reviewer 2: ```suggestion
> **Note:** The tag `Severity` has special meaning in the [VictorOps Incident Fields](https://help.victorops.com/knowledge-base/incident-fields-glossary/). If an alert panel defines this key, then it replaces the `message_type` in the root of the event sent to VictorOps.
```
